app/assets/javascripts/Components/repo_browser.jsx

```diff
@@ -14,6 +15,7 @@ class RepoBrowser extends React.Component {
 
   componentDidMount() {
     this.fetchRevisions();
+    this.submissionFileManagerRef.current.fetchData();
```

Comment:
This should no longer be necessary if you make the change I suggested above. The file manager already has a `componentDidUpdate` hook that calls `fetchData` when its revision identifier prop changes (which should be triggered by a state change in `fetchRevisions`).